src/app/form/form.component.html

```diff
@@ -0,0 +1,22 @@
+<!-- Bootstrap Link -->
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css"
+  integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+
+<script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.6.9/angular.min.js"></script>
+<div class="container">
+  <div class="row">
+    <div class="col-xs-12">
+      <h1> Assignment-2</h1>
+
+      <div class="form">
+
+        <p>Hello, Plz Enter Your Name:</p>
+        <p>UserName: <input type="text" ng-model="name" (input)="submit($event)"></p>
```

Comment:
This is not an implementation for two-way data binding you need to use [(ngModel)] instead
Check this doc https://angular.io/guide/forms

Reply:
done, Thank you.